users/class_tests/constants.py

```diff
@@ -0,0 +1,22 @@
+VALID_PASSWORD = 'pw123123'
+VALID_EMAIL = 'valid@mta.ac.il'
+VALID_USERNAME = 'valid_username'
+VALID_USERNAME2 = 'valid_username2'
+
+NOT_VALID_PASSWORD = 'pw123123'
```

Comment:
Could you please elaborate on how the same password is defined as valid in line # 1 and invalid here?

Reply:
my mistake. 
change to password